javascript.md

```diff
@@ -1241,3 +1434,44 @@
   ```
   </p>
   </details>
+
+-
+  <details>
+    <summary>
+      <b>A21.</b> Cyclomatic complexity SHOULD be less than 5 and MUST be less than 7
+    </summary>
+  <p>
+
+  Bad:
+
+  ```typescript
+    const maxOfThree = (x, y, z) => {
+    if(!isNaN(x) && !isNaN(y) && !isNaN(z)){
+        if(x > y){
+          if(x > z){
+            return x;
+          } else if(y > z){
+            return y;
+          } else {
+            return z;
+          }
+        }
+      } else {
+        throw new Error('x, y, and z MUST be numbers');
+      }
+    }
+  ```
+
+  Good:
+  ```typescript
+    const maxOfThree = (x, y, z) => {
+    const triplet = [x, y, z];
+    if(triplet.some(isNaN)){
+      throw new Error('x, y, and z MUST be numbers');
+    }
+    return Math.max(...triplet);
+    //OR
+    return triplet.reduce((a, b) => a > b ? a : b);
```

Comment:
```suggestion
    const triplets = [x, y, z];
    if(triplet.some(isNaN)){
      throw new Error('x, y, and z MUST be numbers');
    }
    return Math.max(...triplets);
    //OR
    return triplets.reduce((a, b) => a > b ? a : b);
```
Or something better. Check B5

Reply:
I would say `triplet` is an ok name here. It's pretty clear(although somewhat violates B9).
const couple = [1, 2] //one couple(group of two elements)
const couples = [[1,2], [3, 4]]//array of couples(groups of two elements)
const triplet = [1, 2, 3]//one triplet(group of three elements)
const triplets = [[1, 2, 3], [4, 5, 6]]//two triplets(groups of three elements)